valgrind_suppressions.txt

```diff
@@ -98,3 +97,15 @@
    fun:*QQmlTypeLoader*loadWithCachedUnit*QQmlDataBlob*QQmlPrivate*CachedQmlUnit*Mode*
    ...
 }
+{
```

Comment:
i think the one about should be able to handle this too if it was made more generic ?

If you remove the `fun:*QQmlTypeLoader*loadWithCachedUnit*QQmlDataBlob*QQmlPrivate*CachedQmlUnit*Mode*` line from the one above (`qt_6_register_string_qstring`) then i think this would cover both cases ?

Reply:
good call